### ISSUE TITLE
Pop up menus (“My Projects”, “My Account”) should be keyboard accessible

### ISSUE THREAD

**Issue by phacks** [theodo/falco#147]
Thanks @AlbericTrancart for the report!

There’s currently no way to access those menus when using only the keyboard. This effectively shuts down keyboard-only users from many of Falco’s features, such as having multiple projects.

**Comment by fargito** [comment]
@phacks I think this should be closed as it has been handled in https://github.com/theodo/falco/pull/161
